src/lib/molecules/ExtendedInput/index.js

```diff
@@ -257,7 +257,7 @@ const ExtendedInput = forwardRef((props, ref) => {
     );
 
     return (
-        <Tooltip position="bottom" title={showTooltip ? errorText || tooltipText : ''}>
+        <Tooltip position="bottom" title={tooltipText && isValid ? tooltipText : ''}>
```

Comment:
What if the input is not valid but i need to show some tooltip for any cases.
The tooltip must not depend from validation.
```suggestion
        <Tooltip position="bottom" title={tooltipText ? tooltipText : ''}>
```